index.js

```diff
@@ -219,6 +265,72 @@ ShareDbMongo.prototype.commit = function(collectionName, id, op, snapshot, optio
   });
 };
 
+ShareDbMongo.prototype.startTransaction = function(transactionId, callback) {
+  if (typeof transactionId !== 'string') throw new Error('Invalid Transaction ID');
+
+  if (this._sessions[transactionId]) {
+    // TODO: Proper error code
+    return callback(new Error('Transaction already in progress'));
+  }
+
+  this._transactionOpLinks[transactionId] = [];
+  var session = this._sessions[transactionId] = this._mongoClient.startSession();
+  session.startTransaction();
```

Comment:
One thing that possibly should be noted in docs, is that mongo db transactions require replica sets. They won't work on single instance.